pkgs/applications/audio/audio-recorder/default.nix

```
@@ -1,5 +1,5 @@
 { stdenv, fetchurl
-, pkgconfig, intltool, gnome3
+, pkgconfig, intltool, glib
```

There already is a `glib` attribute.

```suggestion
, pkgconfig, intltool
```